app/javascript/elm/src/LabelsInput.elm

```diff
@@ -74,13 +74,13 @@ update msg model toCmd =
 -- VIEW
 
 
-view : Model -> String -> String -> String -> Bool -> TooltipText -> Html Msg
-view model text_ inputId placeholderText isDisabled tooltipText =
+view : Model -> String -> String -> String -> String -> Bool -> TooltipText -> Html Msg
+view model text_ inputId jsClass placeholderText isDisabled tooltipText =
```

Comment:
What do you think of a more specific name? Something like `inputClass` or even more specific?

Reply:
Also, am I mistaken or `jsClass` is always `"js--profile-names-input"`?

Reply:
> What do you think of a more specific name? Something like `inputClass` or even more specific?

I wanted to underline that this is a dedicated class used by js. the fact that it's an input is not that important here. do you have a suggestion what name to use to make this clearer and more specific?

Reply:
I see the value of that in the css class string (e.g. js--blah) cause it's a marker to say that the class is used in code and not for styling purposes. In here `js` is a bit more blurry cause we are already in js. Unfortunately, I'm not sure I can suggest anything.

Feel free to keep it the way it is :)